### PR TITLE
update license to CC0-1.0

### PR DESCRIPTION
**Pull request by jlovejoy**
update license to CC0-1.0 as per discussion on Sept 20th call and ok'd by contributors

### CODE REVIEW

**Comment by jlovejoy** [comment]
this is the PR related to #683 
asked for review by copyright holders / contributors to these two file: @goneall  and @wking 
also posted on mailing list for review by Oct 18th before merging for 3.3 release

**Comment by jlovejoy** [comment]
for the xsd file - is there any reason not to use "SPDX-License-Identifier: CC0-1.0" there as well for the notation??

**Comment by swinslow** [comment]
Hi @jlovejoy -- with these updates, I'd also suggest updating line 33 in /package.json to change it from:

`"license": "CC-BY-3.0 AND MIT"`

to:

`"license": "CC-BY-3.0 AND CC0-1.0"`

since I think validate-schema.js is the only file under MIT, and it's changing to CC0-1.0 in this PR. 

I think keeping CC-BY-3.0 in the package-level metadata makes sense, since it's referring to the spec's own license.

**Comment by wking** [comment]
+1 to updating `package.json` in this PR, so it's clear what's feeding that value.  @goneall, do you remember why you included CC-BY-3.0 in 378fe01b48e5?  My current guess is that it was supposed to match `ListedLicense.xsd`'s CC-BY-4.0, in which case this PR should update it to:

```json
"license": "CC0-1.0"
```

> I think keeping CC-BY-3.0 in the package-level metadata makes sense, since it's referring to the spec's own license.

But [the spec][1] isn't part of this repository, so we don't need to worry about its license here.

[1]: https://github.com/spdx/spdx-spec

**Comment by goneall** [comment]
> do you remember why you included CC-BY-3.0 in 378fe01?

I don't recall, but changing it to match the current XSD license makes sense to me since it is part of this repo.

**Comment by jlovejoy** [comment]
thanks @swinslow  @wking  and @goneall - I think we want consistency here with CC0-1.0 and to avoid conjunctive (or disjunctive) licensing unless there is work included under a pre-existing license, which I don't think there is here.
I'll do another pass and add any other files to this PR along those lines to be thorough.

**Comment by swinslow** [comment]
Agreed -- thanks all, given the above I agree with just listing CC0-1.0 as the license in the metadata.

**Comment by jlovejoy** [comment]
done. that leaves the only non-xml files in the repo to not have explicit license info to be:

- https://github.com/spdx/license-list-XML/blob/master/Makefile

- https://github.com/spdx/license-list-XML/blob/master/expected-warnings

- goneall.gpg
(plus CONTRIBUTING and README, of course)

I think we are good to merge, then?

**Comment by wking** [comment]
Why don't we do the `Makefile` too?  @goneall and I are the only current contributors there, so we can probably drop it into the new `CC0-1.0` bucket.

File: schema/ListedLicense.xsd [hunk 1]
```diff
@@ -7,7 +7,7 @@
 				This format is internal to the SPDX legal team and is subject to change.  For stable output formats, see <xhtml:a href="https://github.com/spdx/license-list-data">license-list-data</xhtml:a>.
 			</xhtml:p>
 			<xhtml:p>
-				Licensed under CC-BY-4.0.
+				SPDX-License-Identifier: CC0-1.0.
```
Comment (wking) [inline schema/ListedLicense.xsd]: The trailing period here is not part of a valid license expression.  Can we drop it?

Comment (jlovejoy) [reply schema/ListedLicense.xsd]: fixed, good catch!

**Comment by wking** [comment]
> ... where should I put "SPDX-License-Identifier: CC0-1.0"

As the first line?  Do we want to include a Copyright declaration up there too, or are we comfortable leaving that implicit in the version-control history?

> ... and is there specific comment notation that should be used here?

`Makefile`s use [`#` comments][1], so just:

```Makefile
# SPDX-License-Identifier: CC0-1.0
```

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html#tag_20_76_13_01

**Pull request merged**